microsoft-edge/devtools-guide-chromium/index.md

```diff
@@ -139,13 +139,13 @@ In the Tools tab group, you may open the different tools that are available in t
 The DevTools provides lots of features and functionality to use with your web product.  Access the different parts of the DevTools in many ways, but the fastest way to access the features you need is to use the command menu.  For more information, navigate to [Run commands with the Microsoft Edge DevTools Command menu][DevtoolsGuideCommandMenuIndex].  To open the command menu, complete one of the following actions.  
```

Comment:
"complete one of the following actions" is robot words; unnatural.  
https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice#write-like-you-speak

Reply:
I know you have a work item planned to address problems in this article. So I'll leave this as is for now and will only focus on changing screenshots from now on.